### ISSUE TITLE
Track read status

### ISSUE THREAD

**Issue by burntcookie90** [dinosaurwithakatana/hacker-news-android#60]

**Comment by burntcookie90** [comment]
Best way to do this? Maybe have a `Read` table in the DB?

**Comment by nguyenmp** [comment]
Might not be my place to say but I'm working on a similar app for a different website.  I ended up implementing a `LinkMetadata` table which looks something like this:
````
LinkMetadata:
    StoryID
    ReadStatus
    HiddenStatus
    LastReadCommentCount
````

**Comment by burntcookie90** [comment]
Nope, that's legit, I like that idea! This will allow flexibility for future story statuses

**Comment by burntcookie90** [comment]
Though it may be smarter to store this in a cache, perhaps using DiskLruCache
